libraries/radpdfprocessing/model/form.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Form
+page_title: Form
+description: Form
+slug: radpdfprocessing-model-form
+tags: form
+published: True
+position: 6
+---
+
+# Form
+
+
+
+__Form__ is a content element, which contains a form source and represents a Form XObject. The Form XObjects enables you to describe objects (text, images, vector elements, etc.) within a PDF file and reuse this content among the document.
+      
+
+* [Creating and Inserting a Form](#creating-and-inserting-a-form)
+
+* [Form Properties](#form-properties)
+
+## Creating and Inserting a Form
+
+The **Form** class exposes a default public constructor to allow you create instances of it. __Form__ is a content element and you can add such an object to the __Content__ collection of an __IContainerElement__ such as [RadFixedPage]({%slug radpdfprocessing-model-radfixedpage%}).
```

Comment:
Do we really need the first sentence? "The **Form** class exposes a default public constructor to allow you create instances of it. "